spec/config.json

```
@@ -201,6 +216,41 @@
       "examples": [
         "1h"
       ]
+    },
+    "tls_config": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Configures HTTPS (HTTP over TLS). If configured, the server automatically supports HTTP/2.",
+      "properties": {
+        "strict": {
+          "type": "boolean",
+          "description": "Disallow any plaintext (HTTP) traffic and force HTTPS.",
+          "default": true
+        },
```

Rename to `enabled` and only allow this to be used in the admin endpoint

```suggestion
        "enabled": {
          "type": "boolean",
          "description": "Setting enabled to false drops the TLS requirement for the admin endpoint, even if TLS is enabled on the public endpoint.",
          "default": true
        },
```